docs/creature_template.md

```diff
@@ -27,11 +27,14 @@ This table contains the description of creatures. Each spawned creature is an in
 | [speed\_run](#creature_template-speed_run)                        | float                 | NO   |     | 1.14286 |       | Result of 8.0/7.0, most common value |
 | [scale](#creature_template-scale)                                 | float                 | NO   |     | 1       |       |                                      |
 | [rank](#creature_template-rank)                                   | tinyint(3) unsigned   | NO   |     | 0       |       |                                      |
+| [mindmg](#creature_template-mindmg)                               | float                 | NO   |     | 0       |       |                                      |
+| [maxdmg](#creature_template-maxdmg)                               | float                 | NO   |     | 0       |       |                                      |
 | [dmgschool](#creature_template-dmgschool)                         | tinyint(4)            | NO   |     | 0       |       |                                      |
+| [attackpower](#creature_template-attackpower)                     | int(10) unsigned      | NO   |     | 0       |       |                                      |
 | [BaseAttackTime](#creature_template-BaseAttackTime)               | int(10) unsigned      | NO   |     | 0       |       |                                      |
 | [RangeAttackTime](#creature_template-RangeAttackTime)             | int(10) unsigned      | NO   |     | 0       |       |                                      |
-| [BaseVariance](#creature_template-BaseVariance)                   | float                 | NO   |     | 1       |       |                                      |
-| [RangeVariance](#creature_template-RangeVariance)                 | float                 | NO   |     | 1       |       |                                      |
+| BaseVariance                                                      | float                 | NO   |     | 1       |       |                                      |
+| RangeVariance                                                     | float                 | NO   |     | 1       |       |                                      |
```

Comment:
there are no such fields in AC, so we should delete them completely

Reply:
Alright, didn't wanted to delete them completely, but if you ask for I can do it hahaha